website/blog/2017-12-18-jest-22.md

```diff
@@ -0,0 +1,118 @@
+
+1. **Use expect and jest-mock in the browser: **[Michael Jackson](https://github.com/mjackson) donated his excellent `[expect](https://github.com/mjackson/expect)` package to the Jest project. As part of that transition, the Jest core team, with much help from [Kenneth Skovhus](https://github.com/skovhus/), made both `jest-matchers` (renamed to `expect`) and `jest-mock` work in browsers. This means that while you cannot use Jest itself in browsers ([yet](https://github.com/facebook/jest/issues/848)), you can use its awesome assertions and mocks for instance as replacements for Chai and Sinon running in Mocha tests. If you are migrating from earlier `expect` to the new Jest-powered `expect`, you can use[ jest-codemods](https://github.com/skovhus/jest-codemods/) to automate the migration.
+2. **MIT License:** We changed Jest's license to MIT. *Yay!*
+3. **Fail test suites on async errors:** Jest used to have a bug that made it crash when errors were thrown in certain parts of async code. This was fixed by community contributors.
+4. **Faster startup:** With Jest 21 we fine tuned Jest's startup to be more than 50% faster. The large overhead of Jest when running it on a small and fast test was always an issue for us and now this shouldn't be a reason to hold you back from using Jest any longer.
+
+## Jest Community
+
+The community around Jest is working hard to make the testing experience even greater. These are separate projects from the main Jest project, but we want to highlight some of our personal favorites here.
+
+* [jest-image-snapshot](https://github.com/americanexpress/jest-image-snapshot) – custom matcher to compare images with snapshots by American Express developers
+* [ts-jest](https://github.com/kulshekhar/ts-jest) – all you need to successfully run Jest within TypeScript project by [@kulshekhar](https://github.com/kulshekhar/ts-jest)
+* [jest-codemods](https://github.com/skovhus/jest-codemods/) – migrate your tests from other frameworks to Jest with ease
+* [jest-plugins](https://github.com/negativetwelve/jest-plugins) – a new community project oriented around simplifying setting up test environment for specific tools, like React, or providing some handy utilities
+
+We'd also like to announce that recently we launched a new place for high quality Jest additions – [jest-community](https://github.com/jest-community). It's a new GitHub repository already featuring our favorite projects, like [vscode-jest](https://github.com/jest-community/vscode-jest), [jest-extended](https://github.com/jest-community/jest-extended), to name a few, curated by Jest maintainers and collaborators. We've even migrated our [eslint-plugin-jest](https://github.com/jest-community/eslint-plugin-jest) there, and already see some great contributions, which are published independently at a faster pace. 
+undefined
```

Comment:
wat

Reply:
This should be an image btw

Reply:
makes sense, thanks!

Weird handling though, I'd expect it to be "IMAGE SHOULD BE HERE" or something, not "undefined" :P